monkestation/code/modules/mob/mob_defines.dm

```diff
@@ -2,5 +2,8 @@
 	//MonkeStation Edit: Emote Hotkey Cooldowns
 	var/emote_cooling_down = FALSE
 
+	/// circuit goggle stuff based on research goggle stuff
+	var/circuit_goggles = FALSE
```

Comment:
Going by the other atom/item/mob/human defines, there is no check to see if the one examining is even a mob at all.
This variable should be on the human level, as 99% of mobs can't wear the goggles.